webui/react/src/pages/WorkspaceDetails.tsx

```diff
@@ -94,8 +94,15 @@ const WorkspaceDetails: React.FC = () => {
 
     const response = await getWorkspaceMembers({ nameFilter, workspaceId: id });
     const newGroupIds = new Set<number>();
-    setUsersAssignedDirectly(response.usersAssignedDirectly);
-    setUsersAssignedDirectlyIds(new Set(response.usersAssignedDirectly.map((user) => user.id)));
+    setUsersAssignedDirectly(response.usersAssignedDirectly.filter((u) => u.isActive));
+    setUsersAssignedDirectlyIds(
+      new Set(
+        response.usersAssignedDirectly
+          .filter((u) => u.isActive)
```

Comment:
nit: you could save a loop through the users list by assigning `response.usersAssignedDirectly.filter((u) => u.isActive)` to a variable